internal/internal_test.go

```diff
@@ -380,6 +428,67 @@ func TestCheckDenyObjectDecision(t *testing.T) {
 	}
 }
 
+func TestCheckDenyWithDryRunObjectDecision(t *testing.T) {
+
+	var req ext_authz.CheckRequest
+	if err := util.Unmarshal([]byte(exampleDeniedRequest), &req); err != nil {
+		panic(err)
+	}
+
+	server := testAuthzServerWithObjectDecision(&testPlugin{}, true)
+	ctx := context.Background()
+	output, err := server.Check(ctx, &req)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	fmt.Printf("Result is %v\n", output)
```

Comment:
Remove `fmt.Printf`

Reply:
This was copied from other tests :-p

Removed!

Reply:
Ohh no ! I will take care of `TestCheckDenyObjectDecision `